narayana/ArjunaCore/arjuna/pom.xml

```diff
@@ -85,13 +85,28 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <!-- exclude everything by default then the not-ibmorb and ibmorb profiles can
-                         selectively include/exclude specific tests -->
-                    <excludes>
-                        <exclude>**/*.java</exclude>
-                    </excludes>
-                </configuration>
+                <version>${version.maven.surefire}</version>
+                <executions>
```

Comment:
Do you really want to define two executions where at one all tests are excluded (nothing is run) and the second case is controlled by property?
I can't see any reason in such configuration. But if there is, please elaborate on that.

Reply:
As per Mike's comments below, I will remove the POM changes and will go with different approach to solve this problem